Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,36 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Object Removal Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		if (m_LocalCoreStartupParameter.update)
+		{
+			// Set lock so codes can be enabled/disabled in game without crashes.
+			m_LocalCoreStartupParameter.done = false;
+			for (int current_object_removal_code = 0; current_object_removal_code < m_LocalCoreStartupParameter.num_object_removal_codes; current_object_removal_code++)
+			{
+				for (int d = 0; d < m_LocalCoreStartupParameter.num_object_skip_data_bytes[current_object_removal_code]; d++)
```

Comment:
Ah, that memcmp() suggestion is brilliant!  I just finished implementing it and that whole code loop just got cut down by so much.  Way easier to read now, and probably faster too.  I'm still testing it to make sure there are no problems, but looks good so far.  Takes out the inner for loop and everything inside of it except for the memcmp().

With the new code, if num_objects_skip_data_bytes is > size, it'll still do the comparison anyway and then fail and go to the next code.  I think this is faster than explicitly checking for that case and continue;ing, at least with the new code.